cobalt/loader/loader.cc

```diff
@@ -143,6 +143,11 @@ void Loader::Resume(render_tree::ResourceProvider* resource_provider) {
   if (!is_load_complete_) Start();
 }
 
+void Loader::Conceal() {
```

Comment:
Instead of Plumbing through a new `Conceal`, since `ResourceProviderStub` already signals when the ResourceProvider can't provide graphics resources (See `IsResourceProviderStub` in `image_decoder.cc`, you can instead use the existing `Resume` call to pass that information to all the decoders.

Reply:
I think this would also work as long as there are no issues with changing the resource provider earlier in the WebModule::Impl::Conceal function, as we have to do it before the call to `window_->document()->PurgeCachedResources()`—this felt a little less risky

Reply:
I'll also say there's a lot of value in explicit message passing with clear signals like `Conceal` - it's much more straightforward to reason about and observe the behavior.